meilisearch/index.py

```diff
@@ -2042,6 +2042,68 @@ def reset_proximity_precision(self) -> TaskInfo:
 
         return TaskInfo(**task)
 
+    # LOCALIZED ATTRIBUTES SETTINGS
+
+    def get_localized_attributes(self) -> Union[List[Dict[str, List[str]]], None]:
+        """Get the proximity_precision of the index.
+
+        Returns
+        -------
+        settings:
+            localized_attributes of the index.
+
+        Raises
+        ------
+        MeilisearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+        """
+        return self.http.get(self.__settings_url_for(self.config.paths.localized_attributes))
+
+    def update_localized_attributes(
+        self, body: Union[List[Dict[str, List[str]]], None]
+    ) -> TaskInfo:
```

Comment:
We have tried to move to `Mapping` for these to give more flexablitiy.

```suggestion
    def update_localized_attributes(
        self, body: Union[List[Mapping[str, List[str]]], None]
    ) -> TaskInfo:
```